pennylane_qrack/qrack_device.py

```diff
@@ -195,7 +208,9 @@ def apply_operations(self, operations):
             elif isinstance(op, QubitUnitary):
                 if len(op.wires) > 1:
                     raise DeviceError(
-                        "Operation {} is not supported on a {} device, except for single wires.".format(op.name, self.short_name)
+                        "Operation {} is not supported on a {} device, except for single wires.".format(
```

Comment:
I find "f-strings" to be a bit more Pythonic:

```py
raise DeviceError(f"Operation {op.name} is not supported on a {self.short_name} device, except for single wires.")
```

Reply:
Done.